filament/backend/src/opengl/OpenGLDriver.cpp

```diff
@@ -2192,6 +2192,9 @@ void OpenGLDriver::endRenderPass(int) {
 }
 
 
+void OpenGLDriver::beginSubpass(int) {}
```

Comment:
so this is were glMemoryBarrier would go if we were not using the coherent extension right?

Reply:
Yes, I think so, although Vulkan is the motivation of course. I'm using it both to call vkCmdNextSubpass and to bind the input attachment to a descriptor.   Do you like the name?  Maybe I should just be "nextSubpass" since there's no corresponding end.

Reply:
I agree, I think `nextSubpass` is better, especially that in the future we might support more.